recipes/circrna_finder/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set name = "circrna_finder" %}
+{% set version = "1.2" %}
+{% set sha256 = "2ca764c8c6022af72a9a2366d7b62a6d5d94b4b097a431d3faf381928bbcb7c3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/orzechoj/circRNA_finder/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  host:
+    - perl
+  run:
+    - perl
+    - coreutils
+
+test:
+  commands:
+    - perl --version
```

Comment:
Again, the tests to be be something that verifies the installation has worked. That could be e.g. the `--help` or `--version` of any installed scripts. If they don't those, you can use 'which` to verify the commands are in place.

Reply:
Good to know `which` is an acceptable option. This tool is a relic of a bygone era, there is no help or version parameter 😆